crypto/o_str.c

```diff
@@ -125,6 +125,40 @@ int OPENSSL_hexchar2int(unsigned char c)
           return 0x0E;
     case 'f': case 'F':
           return 0x0F;
+#ifdef CHARSET_EBCDIC
```

Comment:
I'm not sure why you do this here. Would removing the earlier `os_toebcdic` mean that this function just "does the right thing"? Assuming that this function is intended to take a native character set hex digit and convert it to int.

Reply:
This comment seems to be still unresolved.

Reply:
Removed